packages/web-components/fast-element/src/platform.ts

```diff
@@ -79,6 +104,56 @@ if ($global.trustedTypes === void 0) {
     $global.trustedTypes = { createPolicy: (n: string, r: TrustedTypesPolicy) => r };
 }
 
+const propConfig = {
+    configurable: false,
+    enumerable: false,
+    writable: false,
+};
+
+if ($global.FAST === void 0) {
+    Reflect.defineProperty($global, "FAST", {
+        value: Object.create(null),
+        ...propConfig,
+    });
+}
+
+if ($global.FAST.getById === void 0) {
+    const storage = Object.create(null);
+
+    Reflect.defineProperty($global.FAST, "getById", {
+        value<T>(id: string | number, initialize?: () => T): T | null {
+            let found = storage[id];
+
+            if (found === void 0) {
+                found = initialize ? (storage[id] = initialize()) : null;
+            }
+
+            return found;
+        },
+        ...propConfig,
+    });
+}
+
+// TODO: Replace with ??= after TS 4.x update
+($global.FAST.versions ?? (($global.FAST as any).versions = [])).push("1.8.0");
```

Comment:
is this going to be something we need to keep updated?

Reply:
Yep. I looked at a rollup plugin to insert the version but then realized that it obviously doesn't handle the non-rollup situation. So, we might have to write a custom task that just uses regex against the file to shove the package number in there during build or something.

Reply:
Are we going to do that as part of this PR or take as a follow up?

Reply:
I'm going to see if I can maybe get this done tomorrow. I'll ping here for a quick review of that part.

Reply:
@eisenbergeffect just seeing this - I'm going to need to look at the internals of beachball if this is still in question.

Reply:
After some investigation Chris and I discovered that the version injection won't work with beachball. I'm removing this feature for now.